core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerServiceSpec.js

```diff
@@ -0,0 +1,434 @@
+            answer_groups: [
+              {
+                rule_specs: [
+                  {
+                    rule_type: 'Equals',
+                    inputs: {
+                      x: 0
+                    }
+                  }
+                ],
+                outcome: {
+                  dest: 'Question 1',
+                  feedback: [
+                    '<p>y/x is the measure of the slope.</p>'
+                  ],
+                  param_changes: []
+                }
+              }
+            ],
+            confirmed_unclassified_answers: [],
+            customization_args: {
+              choices: {
+                value: [
+                  '<p>y/x</p>',
+                  '<p>-y/x</p>',
+                  '<p>x/y</p>',
+                  '<p>-x/y</p>'
+                ]
+              }
+            },
+            id: 'MultipleChoiceInput'
+          },
+          content: [
+            {
+              type: 'text',
+              value: '<p><b>Linear algebra</b> is a subtopic of maths.</p>'
+            }
+          ],
+          unresolved_answers: {}
+        }
+      }
+    };
+
+    DEFAULT_INTERACTION = {
+      ID: 'MultipleChoiceInput',
+      CUSTOMIZATION_ARGS: {
+        choices: {
+          value: ['<p>Option 1</p>']
+        }
+      }
+    };
+
+    END_EXPLORATION_INTERACTION = {
+      ID: 'EndExploration',
+      CUSTOMIZATION_ARGS: {
+        recommendedExplorationIds: {
+          value: []
+        }
+      }
+    };
+
+    explorationStatesService.init(data.states);
+    explorationTitleService.init(data.title);
+    explorationInitStateNameService.init(data.init_state_name);
+
+    questions = statesToQuestionsService.getQuestions();
+    questionList = questionListObjectFactory.create(questions);
+    lastStateName = questionList.getLastQuestion().getDestinationStateName();
+
+    spyOn(validatorsService, 'isValidEntityName').andReturn(true);
+    spyOn(mockExplorationData, 'autosaveChangeList').andReturn(true);
+  });
+
+  it('should initialize the local data variables', function() {
+    var expectedReturnedValue = simpleEditorManagerService.tryToInit();
+    expect(expectedReturnedValue).toBeTruthy();
+    expect(simpleEditorManagerService.getTitle()).toBe(data.title);
+    expect(simpleEditorManagerService.getIntroductionHtml())
+      .toBe(data.states.Introduction.content[0].value);
+    // 'questionList' is generated from the test data.
+    var expectedSimpleEditorManagerQuestionList = questionList;
+    expect(simpleEditorManagerService.getQuestionList())
+      .toEqual(expectedSimpleEditorManagerQuestionList);
+  });
+
+  it('should return false if initializing the question list fails',
+    function() {
+      var expectedReturnedValue = simpleEditorManagerService.tryToInit();
+      spyOn(statesToQuestionsService, 'getQuestions').andReturn(null);
+      expectedReturnedValue = simpleEditorManagerService.tryToInit();
+      expect(expectedReturnedValue).toBeFalsy();
```

Comment:
Why not toBe(false)?

And you can directly write expect(simpleEditorManagerService.tryToInit()).toBe(false); -- no need for the extra var.